www/include/Administration/parameters/general/form.ihtml

```diff
@@ -105,18 +105,23 @@
 
     <tr class="list_lvl_1">
       <td class="ListColLvl1_name" colspan="2">
-        <h4>{$genOpt_keycloak}</h4>
+        <h4>{$genOpt_openid_connect}</h4>
       </td>
     </tr>
-    <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="keycloak_enable">{$form.keycloak_enable.label}</td><td class="FormRowValue">{$form.keycloak_enable.html}</td></tr>
-    <tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="keycloak_mode">{$form.keycloak_mode.label}</td><td class="FormRowValue">{$form.keycloak_mode.html}</td></tr>
-    <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="keycloak_trusted_clients">{$form.keycloak_trusted_clients.label}</td><td class="FormRowValue">{$form.keycloak_trusted_clients.html}</td></tr>
-    <tr class="list_two"><td class="FormRowField"><img class="helpTooltip" name="keycloak_blacklist_clients">{$form.keycloak_blacklist_clients.label}</td><td class="FormRowValue">{$form.keycloak_blacklist_clients.html}</td></tr>      
-    <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="keycloak_url">{$form.keycloak_url.label}</td><td class="FormRowValue">{$form.keycloak_url.html}</td></tr>
-    <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="keycloak_redirect_url">{$form.keycloak_redirect_url.label}</td><td class="FormRowValue">{$form.keycloak_redirect_url.html}</td></tr>
-    <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="keycloak_realm">{$form.keycloak_realm.label}</td><td class="FormRowValue">{$form.keycloak_realm.html}</td></tr>
-    <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="keycloak_client_id">{$form.keycloak_client_id.label}</td><td class="FormRowValue">{$form.keycloak_client_id.html}</td></tr>
-    <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="keycloak_client_secret">{$form.keycloak_client_secret.label}</td><td class="FormRowValue">{$form.keycloak_client_secret.html}</td></tr>
+    <tr class="list_one"><td class="FormRowField"><img class="helpTooltip" name="openid_connect_enable">{$form.openid_connect_enable.label}</td><td class="FormRowValue">{$form.openid_connect_enable.html}</td></tr>
```

Comment:
for better readability, you could split lines by sub-tags at least by opening <td>